internal/target/koji_target.go

```diff
@@ -18,3 +18,16 @@ func (KojiTargetOptions) isTargetOptions() {}
 func NewKojiTarget(options *KojiTargetOptions) *Target {
 	return newTarget("org.osbuild.koji", options)
 }
+
+type KojiTargetResultOptions struct {
+	HostOS    string `json:"host_os"`
```

Comment:
I agree, it all seems useful.

Reply:
@ondrejbudai I just realized that you probably meant all 4 values from the structure, right?

Reply:
Some values, like the image hash and size are calculated by the code which handles uploading to Koji. Having these values for all osbuild jobs, regardless of the target would require calculating them in a common place for all targets. Totally doable, I'm just pointing this out.

I think that it would be OK to move the `HostOS` and `Arch` to the `OSBuildJobResult`, but I'm inclining to not do it for the `ImageHash` and `ImageSize`. The reason is that as we discussed the possibility of supporting multiple exports and targets, these values would loose their relevancy for the osbuild job as such and would be specific to the target and export (e.g. compressed image will have different values than raw image).

Reply:
You are absolutely right. So let's move only `HostOS` and `Arch` to the results? Does it work for you? (better to do it now so we don't need to care about backward compat later)

Reply:
> You are absolutely right. So let's move only `HostOS` and `Arch` to the results? Does it work for you? (better to do it now so we don't need to care about backward compat later)

I agree!